.github/workflows/coupler.yml

```diff
@@ -0,0 +1,27 @@
+name: Test coupler build
+on: [pull_request]
+
+jobs:
+  coupler-build:
+    runs-on: ubuntu-latest
+    container:
+      image: ryanmulhall/hpc-me.ubuntu-minimal:coupler
+      env:
+        CC: mpicc
+        FC: mpif90
```

Comment:
You have the HPC-ME container, but you aren't using mpiicc and mpiifort.  Is this on purpose?

Reply:
Yeah this is with a gnu based image